knip.production.jsonc

```diff
@@ -1,5 +1,9 @@
 {
-	"$schema": "https://unpkg.com/knip@next/schema.json",
+	"$schema": "https://unpkg.com/knip@latest/schema.json",
 	"entry": ["src/**/index.ts!"],
+	"ignore": [
+		// https://github.com/JoshuaKGoldberg/ts-api-utils/issues/343
```

Comment:
https://github.com/JoshuaKGoldberg/ts-api-utils/pull/341#issuecomment-1868207317 🤷 I couldn't figure it out and don't want to be blocked on it.

Reply:
https://github.com/webpro/knip/issues/414